### DISCUSSION TITLE
Using PHP 8.1 or higher renders warning: undefined array key 0 in /vendor/timber/timber/src/imagehelper.php on line 444

### DISCUSSION THREAD

**Discussion by munts** [flyntwp/flynt#537] [General]
I am staring a new project with FlyntWP 2.1.0 and the latest Premium Components v2.1.1.  When I run npm run build, Everything builds correctly, but I also see, "warning: undefined array key 0 in /vendor/timber/timber/src/imagehelper.php on line 444".  I downgrade to PHP 8.0.22 and the error goes away.  I know that a change was made to finally get rid of the division by 0 error, which is one that I am very happy about, but could that change be causing this?  Or is it just that Flynt or maybe Timber isn't ready for PHP 8.1 and higher? Thanks, Scott

**Comment by timohubois** [comment]
@munts thanks for starting the discussion! 

Think this is may also related to the `sideLoadAvatarImages.php` file, and the warning should only appear once, when an avatar image is side loaded.

You can find some more details here: https://github.com/flyntwp/flynt/discussions/547#discussioncomment-8994649

Hope this helps 🚀
